File: Elixir/fizzBuzz.exs
defmodule FizzBuzz do
  defp _determine(0, 0, _), do: "Fizz Buzz"
  defp _determine(0, _, _), do: "Fizz"
  defp _determine(_, 0, _), do: "Buzz"
  defp _determine(_, _, c), do: c
  def fizz(n) do
    _determine(rem(n, 3), rem(n, 5), n)
  end
end

File: Python/fizzBuzz.py
for i in range(0, 50):
    if i % 3 and i % 5:
        print(str(i) + ' fizz buzz')
    elif i % 3:
        print(str(i) + ' fizz')
    elif i % 5:
        print(str(i) + ' buzz')
    else:
        print(str(i))
